howto_io.md

```diff
@@ -0,0 +1,268 @@
+# How-to: NumPy I/O
+- [Read a .csv or other text file with no missing values](#read-a-csv-or-other-text-file-with-no-missing-values)
+- [Read a .csv or other text file with missing values](#read-a-csv-or-other-text-file-with-missing-values)
+  * [Non-whitespace delimiters](#non-whitespace-delimiters)
+    + [Masked-array output](#masked-array-output)
+    + [Array output](#array-output)
+    + [Array output, specified fill-in value](#array-output-specified-fill-in-value)
+  * [Whitespace-delimited](#whitespace-delimited)
+- [Read a file in .npy or .npz format](#read-a-file-in-npy-or-npz-format)
+- [Write to a file to be read back by NumPy](#write-to-a-file-to-be-read-back-by-numpy)
+  * [Binary](#binary)
+  * [Human-readable](#human-readable)
+  * [Large arrays](#large-arrays)
+- [Read an arbitrarily formatted binary file ("binary blob")](#read-an-arbitrarily-formatted-binary-file-binary-blob)
+- [Write or read very large arrays](#write-or-read-very-large-arrays)
+- [Write files for reading by other (non-NumPy) tools](#write-files-for-reading-by-other-non-numpy-tools)
+- [Write or read a JSON file](#write-or-read-a-json-file)
+- [Save/restore using a pickle file](#saverestore-using-a-pickle-file)
+- [Convert from a pandas DataFrame to a NumPy array](#convert-from-a-pandas-dataframe-to-a-numpy-array)
+- [Save/restore using `numpy.tofile` and `numpy.fromfile`](#saverestore-using-numpytofile-and-numpyfromfile)
+
+## Read a .csv or other text file with no missing values

```

Comment:
Should define the acronym and explain in ten words what missing values are

Reply:
* Should define the acronym

Some acronyms need no introduction. We don't spell out ASCII. `.csv` is widely known even to people who have no idea what NumPy is, and will certainly be known to the person who needs to read a file with a .csv extension. Moreover, csv delimiters aren't always commas, so spelling it out makes it less clear.  

A compromise might be to link to the [Wikipedia article](https://en.wikipedia.org/wiki/Comma-separated_values). In that case I'd follow the Google doc style guide (which I didn't here) and call the format CSV,

* explain in ten words what missing values are

What wording do you think would be helpful? Won't it be clear from the how-to headings that a "missing value" can be either a field missing data so the column count runs short, or a field that contains a special marker indicating the data is unusable?  And won't users reading data files be painfully knowledgeable on the subject of missing data?